src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -177,6 +177,28 @@ bool TuneParamsSelector::VerifyTuneParams(const fully_connected_params& params,
     if (tparams.tile_ofm * simd > 64)
         return false;
 
+    if (tparams.kernel_type == FullyConnected_bf_tiled::KernelType::SLM) {
+        const auto required_batch_alignment = 64;
+        if (!params.is_shape_agnostic && (!IsAligned(output_b, required_batch_alignment) || output_b < 256))
+            return false;
+
+        const auto required_tile_b = 8;
+        if (tparams.tile_b != required_tile_b)
+            return false;
+
+        const auto required_tile_ofm = 2;
+        if (tparams.tile_ofm != required_tile_ofm)
+            return false;
+
+        if (params.weights.GetDType() != WeightsType::INT4 && params.weights.GetDType() != WeightsType::UINT4)
```

Comment:
Do you have a check somewhere that available SLM size is big enough for given parameters? Haven't found one

Reply:
The maximum possible size of SLM allocation of current implementation is 8KB and it looks like all current HW met this requirement. But I will add this condition for clarity, thanks